apple/swift-docc-render
# Change `Frameworks` to `Technologies` to support documenting more product types

Bug/issue #, if applicable: 83451964

## Summary
This PR changes the wording of the side bar from `Frameworks` to `Technologies` to improve the presentation of non-framework documentation. This is in support of some work that has already landed on `main` in `Swift-DocC` and has been discussed on the Swift Forums [here](https://forums.swift.org/t/improve-presentation-of-non-framework-documentation-sr-15434/53388).
I also changed variable names that reference `framework(s)` to `technology(ies)`.

| Before | After |
|--------|-------|
|   <img width="1017" alt="Screen Shot 2021-12-03 at 11 49 06 AM" src="https://user-images.githubusercontent.com/87735557/144670918-595ee613-2237-43b8-a3ca-9d174fa3de0d.png">  |  <img width="980" alt="Screen Shot 2021-12-03 at 11 48 48 AM" src="https://user-images.githubusercontent.com/87735557/144670906-0a999eb4-6a54-4572-a648-2fe8850606b5.png"> |

## Testing
Steps:
Download this fixture file: [fixture.zip](https://github.com/apple/swift-docc-render/files/7652263/fixture.zip)
1. Run VUE_APP_DEV_SERVER_PROXY=/path/to/manual-fixtures/ npm run serve
2. Go to http://localhost:8080/documentation/example
3. Verify that the Summary side bar displays `Technologies` instead of `Frameworks`

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
